client/src/GroupStore.ts

```diff
@@ -0,0 +1,48 @@
+import BaseAnnotationStore, { InsertArgs, MarkChangesPending } from './BaseAnnotationStore';
+import { AnnotationId } from './BaseAnnotation';
+import Group from './Group';
+import MultiMap from './MultiMap';
+
+export default class GroupStore extends BaseAnnotationStore<Group> {
+  // fast reverse mapping of tracks to the collection of groups they are in.
+  trackMap: MultiMap<AnnotationId, AnnotationId>;
```

Comment:
Those are generic params.   A MultiMap is implemented as a mapping of AnnotationId to Set<AnnotationId>.

Reply:
Okay I think I was looking at the map backwards it's really  `MultiMap<GroupId, TrackId>`  Because `MultiMap<trackId, GroupId>` wouldn't allow a track to belong to multiple groups.